certification_organism/__manifest__.py

```diff
@@ -61,7 +60,9 @@
     },
     'author': 'Odoo S.A.',
     "cloc_exclude": [
+        "data/ir_ui_view.xml",
         "data/knowledge_article.xml",
+        "data/website_view.xml",
         "static/src/js/my_tour.js",
     ],
     'images': ['images/main.png'],
```

Comment:
You have to be careful, you are excluding the while ir_ui_view.xml. Thus those are also excluded :

- x_control_charging_station_ir_ui_view_1
- x_control_charging_station_ir_ui_view_2
- x_control_charging_station_ir_ui_view_3